src/Commands.StorSimple.Library/IKeyManager.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
```

Comment:
This `csproj` is used only by StorSimple projects, please merge it into the commands project. Although I fully recognize that this can be independent project from the commands but in Azure PowerShell we already have lot of projects and we are not looking into adding more unnecessary ones just for modularity purposes. If you think this code MUST be like think, think of having it somewhere else and publish a NuGet package for it for getting consumed by Azure PowerShell